app/Module.scala

```diff
@@ -17,9 +17,9 @@
 import com.google.inject.AbstractModule
```

Comment:
Could this be moved into a package and explicitly registered in application.conf? I think it would be better for consistency within this repo and seems to be how we do things in other repos.